dockstore-webservice/src/main/resources/migrations.1.14.0.xml

```diff
@@ -276,4 +276,13 @@
         <addUniqueConstraint columnNames="metricsid" constraintName="uk_metricsid_version_metrics" tableName="version_metrics"/>
         <addForeignKeyConstraint baseColumnNames="metricsid" baseTableName="version_metrics" constraintName="fk_metricsid_version_metrics" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="metrics"/>
     </changeSet>
+    <changeSet id="tighten constraint for names" author="dyuen">
+        <sql dbms="postgresql">
+            CREATE UNIQUE INDEX case_insensitive_toolname on tool using btree (registry, namespace, name, lower(toolname)) where toolname is not null;
+            CREATE UNIQUE INDEX case_insensitive_apptool_toolname on apptool using btree (sourcecontrol, organization, repository, lower(workflowname)) where workflowname is not null;
```

Comment:
In theory it could happen here too between now and the release, couldn't it? Even so, no action needed.

Reply:
Yes, we may need to increment the id. So far, it seems to happen around 1.5-2 months or so? 
No guarantees of course